lib/utils/constants_file.dart

```diff
@@ -0,0 +1,7 @@
+abstract class Constants {
+  static const double image_height = 150;
+  static const int mob_number = 6;
+  static const double fontSize_title = 25;
+  static const double fontSize_text = 40;
```

Comment:
Remember that constants are in lowerCamelCase, so try to write like this: imageHeight, modNumber

Reply:
Done